erpnext/stock/doctype/purchase_receipt/test_purchase_receipt.py

```diff
@@ -414,11 +414,18 @@ def test_purchase_return_partial(self):
 		self.assertEqual(return_pr_2.items[0].qty, -3)
 
 		# Make PI against unreturned amount
+		buying_settings = frappe.get_single("Buying Settings")
```

Comment:
This will become simpler after merging this test util: https://github.com/frappe/erpnext/pull/25805/commits/76dd6e904682a1bec1ff21d66c45e164fd26a47b